import-export-cli/cmd/k8s/genDeploymentDir.go

```diff
@@ -143,10 +146,11 @@ func executeGenDeploymentDirCmd() error {
 			if errUnmarshal != nil {
 				utils.HandleErrorAndExit("Error unmarshal api configmap into struct ", errUnmarshal)
 			}
-			apiCrd.Name = apiMetaData.Name
-			apiCrd.Spec.SwaggerConfigMapName = fmt.Sprintf("%v-cm", apiMetaData.Name)
-			apiCrd.Spec.ParamsValues = fmt.Sprintf("%v-params", apiMetaData.Name)
-			apiCrd.Spec.CertsValues = fmt.Sprintf("%v-certs", apiMetaData.Name)
+			apiName := utils2.GetValidK8sResourceName(apiMetaData.Name)
+			apiCrd.Name = apiName
+			apiCrd.Spec.SwaggerConfigMapName = fmt.Sprintf("%v-cm", apiName)
+			apiCrd.Spec.ParamsValues = fmt.Sprintf("%v-params", apiName)
+			apiCrd.Spec.CertsValues = fmt.Sprintf("%v-certs", apiName)
```

Comment:
Noticed that, this code segment is getting repeated in the code 3 times. Shall we move this to a separate function and reuse it?

Reply:
+1

Reply:
fixed with https://github.com/wso2/product-apim-tooling/pull/703/commits/b01140f8cb4f3c34eca58b05da64c057839cfddd